packages/data-stores/CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## Unreleased
+
+### Breaking changes
+
+- Remove all custom query and mutation hooks (based on `@tanstack/query`) from `support-queries/*`. These are moved to `@automattic/help-center`.
+- Move the following query and mutation hooks (based on `@tanstack/query`) from `queries/*` to `@automattic/help-center`
+  - `use-is-wporg-site.ts`
```

Comment:
Nit: maybe it could be more helpful to list the hook names and not the file names here.